plugins/woocommerce/client/legacy/css/twenty-nineteen.scss

```diff
@@ -883,6 +883,10 @@ table.variations {
 	}
 }
 
+.is-large.wc-block-cart .wc-block-cart__totals-title::before {
+	content: none;
```

Comment:
Remove the top line on the Twenty Nineteen theme.

![Screenshot 2024-10-16 at 15 29 36](https://github.com/user-attachments/assets/6bec45eb-7504-4dc4-ad64-2b38eb0061d2)

Reply:
Thank you for testing this far back